src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(value) {
+      calculator.result += value;
```

Comment:
You should use 'this.result' instead of 'calculator.result' to make sure that 'result' is always accessed from the current instance of the object.